src/main/resources/assets/js/application/recruiter/controllers/tdprWeekTableController.js

```diff
@@ -21,27 +22,20 @@ define(['angular', 'application/recruiter/tdprRecruiterModule', 'application/rec
         $scope.displayedStartDate = $scope.days[0];
         $scope.displayedEndDate = $scope.days[4];
 
-        var offset = 0;
+        $scope.offset = 0;
 
-        var showDataForWeek = function (offset) {
-            $scope.days = tdprDateService.getWeekWithOffset(offset);
+        $scope.changeWeek = function (offset) {
+
+            offset == WeekNavigateEnum.current ? $scope.offset = WeekNavigateEnum.current : $scope.offset += offset;
```

Comment:
Wouldn't
`$scope.offset  = (offset === WeekNavigateEnum.current) ? $scope.offset = WeekNavigateEnum.current : $scope.offset + offset;`
or even if-else be clearer?